samples/function_chaining_custom_status/DurableActivity/__init__.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+def main(name: str) -> str:
+    """Activity function performing a specific step in the chain
+    
+    Parameters
+    ----------
+    trumber : int
+        key to a dictionary that advances the chain
+    
+    Returns
+    -------
+    [int]
+        value in the dictionary as a result
+    """
+
+    switch_statement = {}
+    switch_statement["One"] = "Two"
+    switch_statement["Two"] = "Three"
+    logging.warning(f"Activity Triggered: {name}")

```

Comment:
We should probably also update this logging with something more specific to this demo

Reply:
Will do.

Reply:
In the activity function there is no change.